### PR TITLE
Release 0.10.1

### PR DESCRIPTION
**Pull request by 12wrigja**

### CODE REVIEW

**Comment by 12wrigja** [comment]
@arshaw Once this merged I'll publish a release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 
 # Change Log
 
+## 0.10.1
```
Comment (justingrant) [inline CHANGELOG.md:4]: Won't this release also include the last batch of ported commits from upstream?

Comment (justingrant) [reply CHANGELOG.md:4]: Oh, nvm, I realized that this was just the test runner not the polyfill.

**Review by justingrant — APPROVED** [review]

**Comment by 12wrigja** [comment]
https://www.npmjs.com/package/@js-temporal/temporal-test262-runner

Published.

I'll try and fix #17 this week.

Going forward, it would be nice to have a GH Action check that we update the CHANGELOG alongside any changes...

**Pull request merged**